src/test/resources/configuration-as-code.yml

```diff
@@ -0,0 +1,39 @@
+unclassified:
+  email-ext:
+    defaultSuffix: "@domain.extension"
+    addAccounts:
+      - address: "foo.bar"
+        smtpHost: "smtp-host"
+        smtpPort: "1234"
+        smtpUsername: "smtp-username"
+        smtpPassword: "smtp-password"
+        useSsl: true
+        advProperties: "avd-properties"
+    charset: "UTF-8"
+    defaultContentType: "text/html"
+    defaultSubject: "DEFAULT EMAIL SUBJECT"
+    defaultBody: "DEFAULT BODY"
+    emergencyReroute: "emergency-reroute"
+    maxAttachmentSize: 42
+    mailAccount:
+      address: "foo.bar.xyz"
+      smtpHost: "smtp-host-xyz"
+      smtpPort: "9876"
+      smtpUsername: "smtp-username-xyz"
+      smtpPassword: "smtp-password-xyz"
+      advProperties: "adv-properties-xyz"
+      useSsl: true
+    defaultRecipients: "first-account@domain.extension"
+    allowedDomains: "@domain.extension"
+    excludedCommitters: "not-this-committer"
+    listId: "list-id"
+    precedenceBulk: true
+    defaultReplyTo: "no-reply@domain.extension"
+    adminRequiredForTemplateTesting: true
+    watchingEnabled: true
+    allowUnregisteredEnabled: true
+    defaultPresendScript: "default-presend-script"
+    defaultPostsendScript: "defaultpostsend-script"
+    defaultClasspath: []
+    defaultTriggerIds: []
```

Comment:
FYI, if no trigger ids are specified, then the FailureTrigger will be set (this is for historical reasons). So, if you don't provide anything, make sure the test checks that FailureTrigger is setup in the defaultTriggerIds.

Reply:
I'll have a test with both: some triggers and none. Here it's just because I was in a hurry.. sorry.

Reply:
No worries, I definitely appreciate the help, it was mainly an FYI to make sure you knew before trying to figure out why there was a trigger configured when you setup none :-)